src/test/fuzz/script_sigcache.cpp

```diff
@@ -29,7 +30,7 @@ void initialize_script_sigcache()
 FUZZ_TARGET(script_sigcache, .init = initialize_script_sigcache)
 {
     FuzzedDataProvider fuzzed_data_provider(buffer.data(), buffer.size());
-    SignatureCache signature_cache{DEFAULT_MAX_SIG_CACHE_BYTES / 2};
+    SignatureCache signature_cache{DEFAULT_MAX_SIG_CACHE_BYTES / 2, GetRandHash()};
```

Comment:
why `GetRandHash` instead of `ConsumeUint256`?

Reply:
Looking at this some more, I wonder if the `signature_cache` should be static. Before, the test was re-using the same cache, now it makes a new one on every run. Maybe this should be set up once in `initialize_script_sigcache()`? On the other hand I think it would also be nice to apply your suggestions and test with different nonces and sizes.

Reply:
Decided to just initialize it statically. If someone wants to fuzz the constructor interface, I think that could go into a follow-up.